ledger/acctonline_test.go

```diff
@@ -453,14 +465,14 @@ func TestAcctOnlineRoundParamsCache(t *testing.T) {
 	allTotals := make(map[basics.Round]ledgercore.AccountTotals)
 
 	start := basics.Round(10)
-	end := basics.Round(2*proto.MaxBalLookback + 15)
+	end := basics.Round(2*maxBalLookback + 15)
 	for i := start; i < end; i++ {
-		consensusVersion := protocol.ConsensusV30
-		if i > basics.Round(proto.MaxBalLookback) {
-			consensusVersion = protocol.ConsensusCurrentVersion
+		consensusVersion := testProtocolVersion1
+		if i > basics.Round(maxBalLookback) {
+			consensusVersion = testProtocolVersion2
 		}
-		if i > 2*basics.Round(proto.MaxBalLookback) {
-			consensusVersion = protocol.ConsensusFuture
+		if i > 2*basics.Round(maxBalLookback) {
+			consensusVersion = testProtocolVersion3
 		}
 		rewardLevelDelta := crypto.RandUint64() % 3
```

Comment:
could you give some more pointers where did the overflow happen?
this rewardLevel used to be as max as 655*2 = 1310 as max. How's such a small value can trigger an overflow in 64-bit integer calculations?

Reply:
The overflow happens in `accumulateTotals`. I haven't looked into exactly why this happens, but I know that when I copied your test in acct updates and doubled the number of rounds, it started overflowing.